client/src/components/LogoText.jsx

```diff
@@ -28,30 +33,21 @@ const Logo = styled.img`
   height: 50%;
   transform: translateY(30px) translateX(175px);
   z-index: 5;
-`;
 
-/*     <a style={{width: "60%", height: "0%"}}
-      href="https://chrome.google.com/webstore/detail/focus-pocus-extension/ognhkeempdpgnfkliplegljejeakonlg/"
-      target="_blank"
-    > */
+  @media (max-width: 1300px) {
+   height: 250px;
+   transform: none;
+  }
+`;
 
-const EXTENSION_URL = "https://github.com/LHL-FocusPocus/FocusPocus/releases"
-// const EXTENSION_URL = "https://chrome.google.com/webstore/detail/focus-pocus-extension/ognhkeempdpgnfkliplegljejeakonlg/"
```

Comment:
The extension URL should be the chrome store one since it's working right now. But leave the other one commented out in case it gets pulled from the store so we can switch easily. This needs to be done for `navbar.jsx` too.